File: java/Kendaraan.java
public class Kendaraan {
  private String idKendaraan;
  private String merk;
  private int tahunProduksi;
  private double harga;

  public Kendaraan(String idKendaraan, String merk, int tahunProduksi, double harga) {
    this.idKendaraan = idKendaraan;
    this.merk = merk;
    this.tahunProduksi = tahunProduksi;
    this.harga = harga;
  }

  public double hitungNilaiDepresiasi() {
    return 0;
  }

  public void tampilkanInfo() {
    System.out.println("ID Kendaraan : " + idKendaraan);
    System.out.println("Merk         : " + merk);
    System.out.println("Tahun        : " + tahunProduksi);
    System.out.println("Harga        : Rp " + harga);
  }

  public String getIdKendaraan() {
    return idKendaraan;
  }

  public String getMerk() {
    return merk;
  }

  public int getTahunProduksi() {
    return tahunProduksi;
  }

  public double getHarga() {
    return harga;
  }
}

File: cpp/Kendaraan.cpp
#include <iostream>
#include <string>

using namespace std;

class Kendaraan
{
private:
    string idKendaraan;
    string merk;
    int tahunProduksi;
    double harga;

public:
    Kendaraan(string idKendaraan, string merk, int tahunProduksi, double harga)
    {
        this->idKendaraan = idKendaraan;
        this->merk = merk;
        this->tahunProduksi = tahunProduksi;
        this->harga = harga;
    }

    void tampilkanInfo()
    {
        cout << "ID Kendaraan : " << idKendaraan << "\n";
        cout << "Merk         : " << merk << "\n";
        cout << "Tahun        : " << tahunProduksi << "\n";
        cout << "Harga        : Rp " << harga << "\n";
    }

    string getIdKendaraan()
    {
        return idKendaraan;
    }

    string getMerk()
    {
        return merk;
    }

    int getTahunProduksi()
    {
        return tahunProduksi;
    }

    double getHarga()
    {
        return harga;
    }
};